draft-ietf-taps-interface.md

```diff
@@ -821,8 +821,8 @@ RemoteSpecifier.WithIPv4Address(192.0.2.22)
 A Preconnection Object holds properties reflecting the application's
 requirements and preferences for the transport. These include Selection
 Properties for selecting protocol stacks and paths, as well as Connection
-Properties for configuration of the detailed operation of the selected Protocol
-Stacks.
+Properties and Message Properties for configuration of the detailed operation
+of the selected Protocol Stacks on a per connection and message level.
```

Comment:
Might be nitpicking here, but -- the Message Properties at this time are not configuring the Protocol Stack, they're just setting common defaults for Send, correct?

Reply:
Correct, but there is no implication when the configuration will actually take effect.

Reply:
```suggestion
of the selected Protocol Stacks on a per-Connection and Message level.
```

I agree that @britram is nitpicking  :-)   ... because I think that pointing out that Message Properties only influence the send call will only make this harder to read, at no real benefit.